web/README.md

```diff
@@ -300,6 +300,18 @@ the root of this repository.
 Keep in mind that there should only be a single `yarn.lock` in this repository, here at the top level. If you add packages
 via `yarn workspace <workspace-name> add <package-name>`, it will create a `packages/<package-name>/yarn.lock` file, which should not be checked in.
 
+Dependencies should generally be added to the specific workspaces that use them. 
+For instance, if you need a new dependency for Web UI, add it to `packages/teleport/package.json`. 
+Similarly, if you're adding a new ESLint plugin, include it in `packages/build/package.json`.
+
+However, there are cases where dependencies should be added to the root `package.json`:
+
+1. Project-wide packages, such as `react`.
+2. Web UI Enterprise dependencies (these shouldn't be added to `e/web/teleport/package.json` 
+to avoid generating a different lockfile when e isn't cloned).
+3. Globally run CLI tools, like `prettier`.
```

Comment:
What is "globally run" in this context? 😏 Did you mean packages that provide CLI tools which we'd like to run from the root of the repo?

Reply:
Yes, I will clarify.